### DISCUSSION TITLE
Datetimes are displayed as AM/PM, but I want 24 hour time notation. How?

### DISCUSSION THREAD

**Discussion by woutersamaey** [btcpayserver/btcpayserver#2714] [General]
On my node all datetimes are displayed as AM/PM, but I want 24 hour time notation. How do I change this?
There does not seam to be a setting.

Is it automatically detected somehow?

Next steps: Should we create a setting for this in the user's account?

**Comment by Zaxounette** [comment]
@Kukks can probably confirm where it's detected from, because I have everything displayed with a 24 hour clock in BTCPay Server.
